### PR TITLE
Fix for Python 3

### PR DESCRIPTION
**Pull request by PetrDlouhy**
based on PR #7

### CODE REVIEW

**Comment by Natim** [comment]
This is great, can we add a tests and a test matrix to make sure it works with the supported version of Django?

We probably want to make it explicit on the README too.

**Comment by PetrDlouhy** [comment]
Now it comes with the Travist testing and Coveralls coverage reports. You will have to enable it for the main repository.

**Comment by Natim** [comment]
Apparently as this repositiory is not in an org, I cannot do it myself. @jojax can you do it?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - DJANGO_VERSION='https://github.com/django/django/archive/master.tar.gz'
+python:
+  - "2.7"
+  - "3.4"
+before_script:
+  - pip install coverage coveralls
+  - pip install -q "$DJANGO_VERSION"
+  - python setup.py install
+script:
+  - coverage run manage.py test --settings=settings
```
Comment (Natim) [inline .travis.yml]: When trying to run this locally I get a ``No file to run: 'manage.py'`` error.

Would you mind also adding a tox.ini file?

**Comment by Natim** [comment]
First of all thanks @PetrDlouhy for your work on this.

Should we also address flake8 errors there?

```
django_js_error_hook/urls.py:6:5: E128 continuation line under-indented for visual indent
django_js_error_hook/tests.py:4:1: E302 expected 2 blank lines, found 1
django_js_error_hook/tests.py:6:1: W293 blank line contains whitespace
django_js_error_hook/tests.py:9:80: E501 line too long (139 > 79 characters)
django_js_error_hook/tests.py:15:1: W293 blank line contains whitespace
django_js_error_hook/tests.py:15:1: W391 blank line at end of file
django_js_error_hook/views.py:13:1: E302 expected 2 blank lines, found 1
django_js_error_hook/views.py:21:25: E126 continuation line over-indented for hanging indent
django_js_error_hook/views.py:23:21: E126 continuation line over-indented for hanging indent
django_js_error_hook/views.py:26:1: E302 expected 2 blank lines, found 1
django_js_error_hook/views.py:36:77: E261 at least two spaces before inline comment
```

File: .travis.yml (hunk 1 above)
Comment (Natim) [inline .travis.yml]: Nevermind it just works:

```
$ coverage run manage.py test --settings=settings
Creating test database for alias 'default'...
ERROR Got error: 
	details: Description of the error by the browser javascript engine.
	user: <UNAUTHENTICATED>
..
----------------------------------------------------------------------
Ran 2 tests in 0.030s

OK
Destroying test database for alias 'default'...
```

**Pull request merged**